python/langsmith/evaluation/_arunner.py

```diff
@@ -192,6 +190,24 @@ async def aevaluate(
         ...     )
         ... )  # doctest: +ELLIPSIS
         View the evaluation results for experiment:...
+
+        Using Async evaluators:
+
+        >>> async def helpfulness(run: Run, example: Example):
+        ...     # Row-level evaluator for helpfulness.
+        ...     await asyncio.sleep(0.1)  # Replace with your LLM API call
+        ...     return {"score": run.outputs["output"] == "Yes"}
+
+        >>> results = asyncio.run(
+        ...     aevaluate(
+        ...         apredict,
+        ...         data=dataset_name,
+        ...         evaluators=[helpfulness],
+        ...         summary_evaluators=[precision],
+        ...         experiment_prefix="My Helpful Experiment",
+        ...     )
+        ... )  # doctest: +ELLIPSIS
+        View the evaluation results for experiment:...
```

Comment:
nit: use await instead of `asyncio.run`

Reply:
That breaks doctests since this is in the python repl